GhostNetwork.Publications.Domain/ICommentStorage.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace GhostNetwork.Publications.Domain
+{
+    public interface ICommentStorage
+    {
+        Task<Comment> FindOneByIdAsync(string id);
+
+        Task<string> InsertOneAsync(Comment publication);
+
+        Task<IEnumerable<Comment>> FindManyAsync(string publicationId, int skip, int take);
+
+        Task<bool> FindCommentInPublicationById(string commentId, string publicationId);
```

Comment:
Don't forget Async suffix
Prefix "Find" imply return model as result. Please change return type or change method name to IsCommentInPublication